web/src/settings_config.ts

```diff
@@ -334,7 +334,13 @@ export const get_stream_topics_policy_values = (): StreamTopicsPolicyValues => {
     return {
         inherit: {
             code: "inherit",
-            description: $t({defaultMessage: "Organization default"}),
+            description: $t(
+                {defaultMessage: "Organization default ({org_level_topics_policy})"},
+                {
+                    org_level_topics_policy:
+                        get_realm_topics_policy_values()[realm.realm_topics_policy]?.description,
```

Comment:
`get_realm_topics_policy_values` is already called above, so that variable can be reused and maybe we can have `!` instead of `?` as we can be sure that `realm.realm_topics_policy` will always be a valid key for `realm_topics_policy_values`.